docs/DeveloperGuide.md

```diff
@@ -120,10 +128,20 @@ To add new entries, user will have to input the related commands.
 Wiagi will then parse the command to the AddCommand class.
 The AddCommand class will then validate the user's input and add the input to IncomeList or SpendingList
 
-#### Deleting of entry, editing of entry
-The commands are similar where there would be a parsing of command to each of its individual classes.
-A similar validation process takes place and actions would be made on IncomeList or SpendingList accordingly
-(deleting entry from list for delete and editing of entry from list for edit)
+#### Editing entries
+The EditCommand validates and parses the given input to determine if it is editing a spending or an income. It then
+extracts the entry from either the respective list(SpendingList or IncomeList). Finally, it uses the parsed input to
+determine which attribute to edit and sets this attribute of the extracted entry to the new value.
+
+![editCommandSequence.png](./Diagrams/Commands/editCommandSequence.png)
```

Comment:
The alt frame should have dotted line in between

Reply:
Fixed!